<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mode_AI</class>
 <widget class="QWidget" name="Mode_AI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI 演示模式</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #050510;</string>
  </property>
  <widget class="QWidget" name="topBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#111;border-bottom:2px solid #00e5ff;</string>
   </property>
   <layout class="QHBoxLayout" name="topLay">
    <property name="rightMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="styleSheet">
       <string notr="true">color:#00e5ff;font:700 14pt 'Microsoft YaHei';</string>
      </property>
      <property name="text">
       <string>AI AUTO PLAY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCountdown">
      <property name="styleSheet">
       <string notr="true">color:#00e5ff;font:700 24pt 'Microsoft YaHei';</string>
      </property>
      <property name="text">
       <string>99:99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelScore">
      <property name="styleSheet">
       <string notr="true">color:#fff;font:700 20pt 'Microsoft YaHei';</string>
      </property>
      <property name="text">
       <string>Score: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="boardWidget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#0a1520;border:2px solid #00e5ff;border-radius:12px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>540</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QPushButton{
      color:#fff;
      font:14pt 'Microsoft YaHei';
      border:1px solid #00e5ff;
      border-radius:8px;
      background:rgba(0, 229, 255, 30);
     }
     QPushButton:hover{background:rgba(0, 229, 255, 60);}
     QPushButton:pressed{background:rgba(0, 229, 255, 100);}
    </string>
   </property>
   <property name="text">
    <string>退出演示</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
